src/box/lua/schema.lua

```diff
@@ -248,7 +248,9 @@ end
  @param table - table with parameters
  @param template - table with expected types of expected parameters
   type could be comma separated string with lua types (number, string etc),
-  or 'any' if any type is allowed
+  or 'any' if any type is allowed. Instead of this string function, which will
+  be used to check if the parameter is correct, can be used too. It should
```

Comment:
Nit: Let's add a comma and an article to make this sentence more readable: Instead of this string, a function, which will be used ...